cypress/integration/chi-vue/tooltip.spec.js

```diff
@@ -0,0 +1,150 @@
+const TOOLTIP_CLASSES = {
+  TOOLTIP_ELEMENT: 'chi-tooltip'
+};
+const TOOLTIP_EVENTS = {
+  SHOW: 'chiTooltipShow',
+  HIDE: 'chiTooltipHide',
+  SHOWN: 'chiTooltipShown',
+  HIDDEN: 'chiTooltipHidden'
+};
+const ACTIVE_CLASS = '-active';
+
+const hasClassAssertion = (el, value) => {
+  cy.get(el).should('have.class', value);
+};
+
+describe('Pagination', () => {
+  before(() => {
+    cy.visit('tests/chi-vue/tooltip.html');
+  });
+
+  describe('Base', () => {
+    beforeEach(() => {
+      cy.get("[data-cy='tooltip-base']")
+        .children()
+        .last()
```

Comment:
For now, this is fine but we will need to change the way chi-tooltip-element is created. Currently, it gets rendered in the same wrapper as the Tooltip trigger while chi.js implementation of Tooltip injects tooltip elements at the end of body (for better positioning, sizing, etc).
@jllr We would need to create a separate task for it.